internal/pkg/manifest/validate.go

```diff
@@ -2226,6 +2226,12 @@ func validateAndPopulateNLBListenerPorts(listener NetworkLoadBalancerListener, p
 	if err != nil {
 		return err
 	}
+
+	// Handle termination of TLS protocol
+	if strings.ToUpper(aws.StringValue(nlbProtocol)) == TLS {
+		nlbProtocol = aws.String(TCP)
+	}
```

Comment:
I think we should move this logic to somewhere around [here](https://github.com/aws/copilot-cli/pull/5549/files#diff-5fc13795122fe030799a09ef956a8b66cec3d40bf1a81266323edbf3626b5685R2248) - `nlbProtocol` can be `TLS`, it's just it doesn't make sense to make `targetProtocol` to be `TLS` because we always terminate `TLS`.